src/services/tx/tx-sender/sdk.ts

```diff
@@ -29,3 +33,40 @@ export const getUncheckedSafeSDK = (provider: Web3Provider): Promise<Safe> => {
 
   return sdk.connect({ ethAdapter })
 }
+
+type SigningMethods = Parameters<Safe['signTransaction']>[1]
+
+export const _getSupportedSigningMethods = (safeVersion: SafeInfo['version']): SigningMethods[] => {
+  const SAFE_VERSION_SUPPORTS_ETH_SIGN = '>=1.1.0'
+
+  if (!safeVersion || !semverSatisfies(safeVersion, SAFE_VERSION_SUPPORTS_ETH_SIGN)) {
```

Comment:
What I don't get tho, why do earlier Safe versions support `eth_signTypedData` (AFAIU a newer EIP), but not `eth_sign`? Or is `eth_sign` newer?

Reply:
[`eth_sign` wasn't added to the contracts until v1.1.0](https://github.com/safe-global/safe-contracts/pull/127) as EIP-712 is/was the preferred signing method.

Reply:
I'm also a bit puzzled by that. How did you find that out?
I cannot really see the related change [here](https://github.com/safe-global/safe-contracts/releases/tag/v1.1.0).

Reply:
That's a nice tidbit but I still don't understand why eth_sign wasn't supported and eth_signTypedData was.

Reply:
I think it was supported, it's more that nowadays most wallets prepend the EIP-191 prefix to `eth_sign` messages and signing with prefix wasn't supported.

Reply:
Original EIP-191 was supposed to be implemented in a separate `personal_sign` method afaik: https://eips.ethereum.org/EIPS/eip-191

Some history (in metamask's interpretation): https://docs.metamask.io/guide/signing-data.html#a-brief-history

Reply:
Thanks @mikhailxyz!

Btw, I was told MM are going to turn off eth_sign by default.